spec/features/authentication/sign_up_spec.rb

```diff
@@ -26,7 +26,7 @@
     within("form") do
       # Errors show below the inputs, Capybara will show the error as follows
       expect(page).to have_content("Emailcan't be blank")
-      expect(page).to have_content("Passwordcan't be blank")
+      expect(page).to have_content("PasswordMinimum is #{Rails.configuration.devise.password_length.min} characterscan't be blank")
```

Comment:
It's because of the way capybara reads from the HTML

That corresponds to html like:

```
<p>Password</p><p>Minimum is 8 characters</p>
```

Capybara just ignores the <p> tags which is why it gets concatenated.